.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,185 @@
+
+1.  Create or log into your development environment account on GitHub.
+
+1.  Navigate to the <a href="https://GitHub.com/magento/devdocs" target="\_blank">DevDocs repository</a>.
+
+1.  Click **Fork** at the top right.
+
+1.  Clone the repository into your development environment and start writing. Commit changes and  and committing your changes. Optionally, create a branch first if you plan to work on multiple changes.
+
+1.  Before opening a pull request, don't forget to review your work for typos, formatting errors, code sample formatting, or sentences that need clarifying.
+
+### Preview HTML locally
+
+To preview your changes in HTML output, follow the instructions in the [README](https://GitHub.com/magento/devdocs/blob/develop/README.md){:target="_blank"} to build the devdocs site locally using Jekyll.
+
+### Update your fork {#sync}
+As we merge changes with the upstream repository, your fork becomes outdated and pull requests might result in merge conflicts.
+To see if your fork is outdated, open the fork page in GitHub. If you see the following message at the top of the page, update your fork: `This branch is <number> commits behind develop`.
+
+There are two ways to update your fork. The typical way is discussed in [GitHub documentation](https://help.GitHub.com/articles/syncing-a-fork){:target="_blank"}. Ensure you are updating your fork from the correct branch.
+
+It is also possible to use the GitHub interface to update your fork. This is referred to as a *reverse pull request*. This method has the downside of inserting unnecessary information into fork commit history.
+
+1.  Navigate to the GitHub page for your forked repository and click **New pull request**. You should see the following message:
+
+        There isn’t anything to compare.
+        magento:2.1 is up to date with all commits from <your fork>:2.1. Try switching the base for your comparison.
+
+1.  Click the base link and then click **Create pull request**.
+
+1.  Enter a name for your pull request.
+
+1.  Scroll to the bottom of the page and click **Merge pull request**.
+
+1.  Click **Confirm Merge**.
+
+## Create a pull request {#pull-request}
+To create a pull request do the following:
+
+1.  Push your changes to your forked repository on GitHub.
+
+1.  In your forked repository, click **New pull request**.
+
+1.  Be sure to create the pull request on the `develop` branch. We do not accept pull requests on other branches, like `gh-pages`.
+
+1.  Review the changes, then click **Create pull request**.
+
+1.  Fill out the form and click **Create pull request** again to submit the pull request&mdash;that’s it!
+
+<div class="bs-callout bs-callout-info" id="info" markdown="1">
+If you have not signed the [Magento Contributor Agreement](http://www.magento.com/legaldocuments/mca), the PR will provide a link to complete. All contributors are required to submit the form and agree to the terms.
+</div>
+
+## Doc templates {#templates}
```

Comment:
Remove custom anchor id and update links to use `#doc-templates`

Reply:
Fixing across